storage/engine/mvcc.go

```diff
@@ -1236,7 +1238,12 @@ func MVCCFindSplitKey(engine Engine, raftID int64, key, endKey proto.Key) (proto
 		done := !bestSplitKey.Equal(encStartKey) && diff > bestSplitDiff
 
 		// Add this key/value to the size scanned so far.
-		sizeSoFar += int64(len(kv.Key) + len(kv.Value))
+		_, _, isValue := MVCCDecodeKey(kv.Key)
+		if isValue {
```

Comment:
The values returned from MVCCDecodeKey are used all over the place... I sometimes think I should reverse the polarity and make it "isMeta" instead. But I don't think a comment here would be too helpful considering the ubiquity of this method invocation.